python/singa/autograd.py

```diff
@@ -725,12 +885,26 @@ def backward(self, dy):
 
 
 def add(a, b):
+    """
+    Performs element-wise binary addition.
+    Args:
+        a (CTensor): First operand..
```

Comment:
a is Tensor or CTensor?

Reply:
sorry, I made a mistake, it should be Tensor, I've checked all the file again to fix it.